src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -154,5 +155,136 @@ public static void VerifyExpiration_LocalTime(DateTime verificationTime, bool sh
                 }
             }
         }
+
+        [Fact]
+        public static void BuildChain_WithApplicationPolicy_Match()
+        {
+            using (var msCer = new X509Certificate2(TestData.MsCertificate))
+            {
+                X509Chain chain = new X509Chain();
```

Comment:
Even though it's a test, we should ideally be disposing of disposable things like this (unless part of the test is not doing so).